modules/build/src/main/scala/scala/build/preprocessing/directives/UsingCompilerPluginDirectiveHandler.scala

```diff
@@ -45,26 +46,29 @@ case object UsingCompilerPluginDirectiveHandler extends UsingDirectiveHandler {
 
   override def keys = Seq("plugin", "plugins")
   override def handleValues(
-    values: Seq[Any],
-    cwd: ScopePath,
-    positionOpt: Option[Position]
+    values: Seq[Value[_]],
+    cwd: ScopePath
   ): Either[BuildException, BuildOptions] = either {
 
     val extraDependencies = value {
       DirectiveUtil.stringValues(values)
-        .map { dep =>
-          // Really necessary? (might already be handled by the coursier-dependency library)
-          val dep0 = dep.filter(!_.isSpaceChar)
+        .map {
+          case (dep, pos) =>
+            // Really necessary? (might already be handled by the coursier-dependency library)
+            val dep0 = dep.filter(!_.isSpaceChar)
 
-          parseDependency(dep0)
+            parseDependency(dep0).map(_ -> pos)
```

Comment:
NIT Build a `Positioned(Seq(pos), _)` right now, so that we don't have to do the same below?